baselines/aad.md

```diff
@@ -17,15 +19,14 @@ hosts the M365 resources being used.
 [home tenant](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/authentication-conditional-access)
 is the one that owns the external user’s (e.g., guest) account.
 
+## Highly Privilged Roles
+
+The following built-in Azure AD roles are considered highly privileged at a minimum: Global Administrator, Privileged Role Administrator, User Administrator, SharePoint Administrator, Exchange Administrator, Hybrid Identity Administrator, Application Administrator, Cloud Application Administrator. Additional built-in roles that are considered highly privileged in the agency's environment can be added to this list.
+
 ## Assumptions
```

Comment:
Why is this an assumption?  Seems it would be a policy?

Reply:
Check out the new 2nd sentence in the Assumptions section to see if the revised wording makes it clearer. In general, on some Microsoft pages they instruct the user to exclude "emergency access" accounts from a conditional access policy. So in the baseline we are saying "we assume you have created emergency accounts".
If it is not clear why we have it there, we might want to remove it from the baseline.

The reason it is not a policy is because we don't dictate a specific process for creating and securing emergency accounts - that is outside the scope of a configuration baseline such as this.